### ISSUE TITLE
Documentation mentions wrong operations not-checked for quorum?

### ISSUE THREAD

**Issue by dreusel** [Netflix/dynomite#782]
The documentation page about consistency (https://github.com/Netflix/dynomite/wiki/Consistency) mentions a list of Redis operations for which there is no quorum check. However two of these commands (MKEYS, MHKEYS) don't seem to exist in the Redis documentation but I do suspect that this limitation applies to the actual HKEYS command, which is not mentioned?

**Comment by ipapapa** [comment]
That is correct, there is a typo. `HMKEYS` and `HKEYS`. @smukil can you please update the wiki?
